kele.el

```diff
@@ -865,6 +865,18 @@ show the requested Kubernetes object manifest.
             (,(kbd "U") . kele--refetch))
   (read-only-mode 1))
 
+(define-derived-mode kele-list-mode tabulated-list-mode "Kele: List"
+  "Major mode for listing multiple resources of a single kind."
+  :group 'kele
+  :interactive nil
+  (setq-local tabulated-list-format
+              (vector (list "Name" 50 t)
+                      (list "Namespace" 20 t)
+                      (list "Group" 10 t)
+                      (list "Version" 5 t)))
```

Comment:
Would love to have the "resource-agnostic columns" here be user-configurable. Let's file a ticket to follow up.

Reply:
See #135.